[2022-06-03 16:21:24,312] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: nyc_bus_breakdown_pipeline.processing_data manual__2022-06-03T16:21:16.955922+00:00 [queued]>
[2022-06-03 16:21:24,323] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: nyc_bus_breakdown_pipeline.processing_data manual__2022-06-03T16:21:16.955922+00:00 [queued]>
[2022-06-03 16:21:24,323] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-03 16:21:24,324] {taskinstance.py:1358} INFO - Starting attempt 1 of 2
[2022-06-03 16:21:24,324] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-03 16:21:24,341] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): processing_data> on 2022-06-03 16:21:16.955922+00:00
[2022-06-03 16:21:24,345] {standard_task_runner.py:52} INFO - Started process 164 to run task
[2022-06-03 16:21:24,349] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'nyc_bus_breakdown_pipeline', 'processing_data', 'manual__2022-06-03T16:21:16.955922+00:00', '--job-id', '135', '--raw', '--subdir', 'DAGS_FOLDER/nycbus_etl.py', '--cfg-path', '/tmp/tmpwm4kni2g', '--error-file', '/tmp/tmp2rot1axp']
[2022-06-03 16:21:24,350] {standard_task_runner.py:80} INFO - Job 135: Subtask processing_data
[2022-06-03 16:21:24,407] {task_command.py:370} INFO - Running <TaskInstance: nyc_bus_breakdown_pipeline.processing_data manual__2022-06-03T16:21:16.955922+00:00 [running]> on host 6cc5764fb927
[2022-06-03 16:21:24,615] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=nyc_bus_breakdown_pipeline
AIRFLOW_CTX_TASK_ID=processing_data
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T16:21:16.955922+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-03T16:21:16.955922+00:00
[2022-06-03 16:21:25,411] {base.py:68} INFO - Using connection ID 's3_conn' for task execution.
[2022-06-03 16:21:25,413] {base_aws.py:210} INFO - Credentials retrieved from extra_config
[2022-06-03 16:21:27,399] {python.py:173} INFO - Done. Returned value was: None
[2022-06-03 16:21:27,420] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=nyc_bus_breakdown_pipeline, task_id=processing_data, execution_date=20220603T162116, start_date=20220603T162124, end_date=20220603T162127
[2022-06-03 16:21:27,454] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-03 16:21:27,504] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
